src/tabs/index.unit.tsx

```diff
@@ -0,0 +1,203 @@
+
+    // Verify the tabpanel is not in the tablist or in a tab.
+    const panelInTabList = tablist.find('[role="tabpanel"]')
+    expect(panelInTabList.exists()).toBe(false)
+    const panelInTabs = tabs.find('[role="tabpanel"]')
+    expect(panelInTabs.exists()).toBe(false)
+
+    // Verify the selected tab ARIA state
+    const activeTab = tablist.find('[role="tab"].kirk-tab-selected')
+    expect(activeTab.exists()).toBe(true)
+    expect(activeTab.length).toBe(1)
+    expect(activeTab.get(0).props['aria-selected']).toBe('true')
+})
+
+it('Should update ARIA attributes after switching tab', () => {
+    const wrapper = mount(
+        <Tabs tabs={defaultTabsConfig.tabs} activeTabId={defaultTabsConfig.activeTabId}></Tabs>,
+    )
+
+    // Verify that initial tab 1 is selected
+    let tabs = wrapper.find('[role="tab"]')
+    expect(tabs.exists()).toBe(true)
+    expect(tabs.length).toBe(3)
+    // Verify aria-selected
+    expect(tabs.get(0).props['aria-selected']).toBe('true')
+    expect(tabs.get(1).props['aria-selected']).toBe('false')
+    expect(tabs.get(2).props['aria-selected']).toBe('false')
+    // Verify tabIndexes
+    expect(tabs.get(0).props['tabIndex']).toBe(0)
+    expect(tabs.get(1).props['tabIndex']).toBe(-1)
+    expect(tabs.get(2).props['tabIndex']).toBe(-1)
+
+    // verify that the current tabpanel id is correct.
+    let tabpanel = wrapper.find('[role="tabpanel"]')
+    expect(tabpanel.get(0).props.id).toBe('tab1_panel')
+
+    // Simulate a click on tab 2.
+    wrapper.find('[role="tab"]').at(1).simulate('click')
+
+    // Verify that now tab 2 is selected
+    tabs = wrapper.find('[role="tab"]')
+    // Verify aria-selected
+    expect(tabs.get(0).props['aria-selected']).toBe('false')
+    expect(tabs.get(1).props['aria-selected']).toBe('true')
+    expect(tabs.get(2).props['aria-selected']).toBe('false')
+    // Verify tabIndexes
+    expect(tabs.get(0).props['tabIndex']).toBe(-1)
+    expect(tabs.get(1).props['tabIndex']).toBe(0)
+    expect(tabs.get(2).props['tabIndex']).toBe(-1)
+
+    // verify that the current tabpanel id is correct.
+    tabpanel = wrapper.find('[role="tabpanel"]')
+    expect(tabpanel.get(0).props.id).toBe('tab2_panel')
+})
+
+it('Should update the panel content', () => {
+    const wrapper = mount(
+        <Tabs tabs={defaultTabsConfig.tabs} activeTabId={defaultTabsConfig.activeTabId}></Tabs>,
+    )
+
+    let tabpanel = wrapper.find('[role="tabpanel"]')
+    expect(tabpanel.text()).toBe('Content for first tab')
+
+    // Simulate a click on tab 2.
+    wrapper.find('[role="tab"]').at(1).simulate('click')
+
+    tabpanel = wrapper.find('[role="tabpanel"]')
+    expect(tabpanel.text()).toBe('Content for second tab')
+
+    // Simulate a click on tab 1.
+    wrapper.find('[role="tab"]').at(0).simulate('click')
+
+    tabpanel = wrapper.find('[role="tabpanel"]')
+    expect(tabpanel.text()).toBe('Content for first tab')
+})
+
+it('Should left/right keyboard navigate', () => {
```

Comment:
nit: you could use a `describe` for your global description then split this test with multiple `it`s

Reply:
Done, thanks!